.gitignore

```diff
@@ -132,7 +131,5 @@ ansible_collections/arista/avd/molecule/*/config_backup
 # Examples config-backup folder
 ansible_collections/arista/avd/examples/*/config_backup
 
-# Ansible test folder
-ansible_collections/arista/avd/tests/
```

Comment:
Any reason for this change in .gitignore as part of this PR?

Reply:
This was a bad commit on my part. Should be fixed now?